src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/VideoBookingHistory.kt

```diff
@@ -0,0 +1,63 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.springframework.data.annotation.Immutable
+import java.time.LocalDate
+import java.time.LocalTime
+
+/**
+ * This entity represents a view projection for v_video_booking_history, and contains the history of a booking.
+ *
+ * It is read-only (immutable).
+ */
+@Entity
+@Immutable
+@Table(name = "v_video_booking_history")
+data class VideoBookingHistory(
```

Comment:
Does the naming of this work?  Would `VideoBookingEvent` be better?

Reply:
I think VideoBookingEvent works better.

Reply:
Done